samples/Samples.HttpClientDriver/HttpClientHelpers.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace;
+
+namespace Samples.HttpClientDriver
+{
+    internal static class HttpClientHelpers
+    {
+        private static readonly Encoding Utf8 = Encoding.UTF8;
+
+        internal static async Task SendHttpClientRequestsAsync(bool tracingDisabled, string url, string requestContent)
+        {
+            // Insert a call to the Tracer.Instance to include an AssemblyRef to Datadog.Trace assembly in the final executable
+            var ins = Tracer.Instance;
```

Comment:
I don't know why I wrote this in but you can delete this line. It's probably best that this project does not have a direct reference to `Datadog.Trace` so it's more representative of a client application